iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HMSTablePropertyHelper.java

```diff
@@ -58,12 +66,21 @@ public class HMSTablePropertyHelper {
       GC_ENABLED, "external.table.purge", TableProperties.PARQUET_COMPRESSION, ParquetOutputFormat.COMPRESSION,
       TableProperties.PARQUET_ROW_GROUP_SIZE_BYTES, ParquetOutputFormat.BLOCK_SIZE);
 
+  public static final Set<String> PROPERTIES_TO_REMOVE = ImmutableSet
```

Comment:
this duplicates the HiveIcebergMetaHook#PROPERTIES_TO_REMOVE

Reply:
Removed from `HiveIcebergMetaHook` and using the ones from `HMSTablePropertyHelper`